Robinhood/Robinhood.py

```diff
@@ -778,3 +920,45 @@ def place_sell_order(
         """
         transaction = Transaction.SELL
         return self.place_order(instrument, quantity, bid_price, transaction)
+
+    ##############################
+    # CANCEL ORDER
+    ##############################
+    def cancel_order(self,oid):
+        """
+        cancel a given order id
+
+        Args:
+            oid (string): the order ID to be cancelled
+        Returns:
+            (:obj:`requests.request`): result from `orders` put command
+            status code 200 signifies success
+
+        """
+        data = {}
+        data['oid'] = oid
+        res = self.session.post(self.endpoints['cancel'].format(**data))
+        res.raise_for_status()
+        return res
+
+
+    def cancel_orders_all(self,instrument=None):
+        """
+        convenience function to cancel all orders, optionally only for a given instrument
+
+        Args:
+            instrument (string): RH instrument URL to restrict results
+        Returns:
+            (:obj:`dict`): containing keys 'cancelled' or 'error' with the list of order ids
+        """
+        orders = self.order_history(instrument=instrument)
+        res = {'cancelled':[], 'error':[]}
+        stoppedOrderStates = (OrderState.FILLED, OrderState.CANCELLED, OrderState.FAILED, OrderState.REJECTED)
```

Comment:
move `stopped_order_states` up to class-global and make it `_stopped_order_states` as a private collection

Reply:
ah, yes, I forgot to move this and make it more generally available.  Given all the States RH api can return I figured we'd want these access to these groupings.  I was thinking convenience  methods could work `isStoppedOrderStatus()`.  Could be added to the Enum and/or through subclassing and using isinstance(state, OrderStopped).  importing the Enums outside the library also feels clumsy right now, I might be missing something tho..
```
from Robinhood.Robinhood import Transaction
```
Should they be within class Robinhood instead?

Reply:
I added a commit to move the Enums and test state.  I think this is a much cleaner approach.